data/json/snippets/e_logs.json

```diff
@@ -74,24 +74,56 @@
+  },
+  {
+    "id": "local_files_simple_25",
+    "text": "untitled2047-1564.mp4\n\nIn what might be the first cheerful voices you’ve heard since the Cataclysm, a muddle of laughs and good-natured conversation fills the recording’s audio.  The view is filled by a small group of grinning college-age individuals crowding in a haphazard line before the camera, a network of tents and camping equipment visible in the gloom-shrouded field behind them.\n\n\"Hey Mallick\" chirps a feminine voice as the film pans across the assembly, a handful of whooped salutations accompanying the speaker’s voice.  \"We all heard about you being laid up in the hospital after the scuffle in <city> last week, and we wanted to say that we're actually really missing your broken ass out here in bum fuck nowhere. We're recording this video cause we wanted to tell you...\"\n\n\"GET WELL SOON!\" As the raucous chorus of disjointed good wishes dies down, the friends' grins rapidly melt away and are replaced by a look of unified confusion as the call... repeats.  An uneasy silence falls across the group as, from an unseen point in the darkness, the chant resounds, growing more distorted and twisted with each inhuman repetition.  A muttered \"What the fuck?\" can be heard as the call continues, seeming to grow louder or closer, now accompanied with the audible crunch of trampled vegetation.  Nervously, a young man draws a 10mm pistol from beneath his jacket and opens his mouth, before a shapeless, pink mass gallops from the gloom behind him on many nimble legs, rapping mauve tendrils about his head as it grips his shoulders with twitching claws, before wrenching its horrific tendrils and ripping the man’s head clean off.\n\n\"GET WELL SOON!\" screams the Mi-go, blood showering from its fleshy, flower-like head before it lunges at the camera."
```

Comment:
add "GET WELL SOON!" to the string of lines migo can say ingame, as a reference to this snippet.